README.md

```diff
@@ -80,10 +80,6 @@ exploring features that contribute to a prediction the model makes.
 It also helps understand which neurons and layers are important for
 model predictions.
 
-Currently, the library uses gradient-based interpretability algorithms
```

Comment:
nit: Could alternatively update this to gradient and perturbation-based, but either way is fine